lib/evss/gi_bill_status/configuration.rb

```diff
@@ -2,15 +2,28 @@
 
 module EVSS
   module GiBillStatus
+    ##
+    # HTTP client configuration for the {GiBillStatus::Service},
+    # sets the base path and a service name for breakers and metrics.
+    #
     class Configuration < EVSS::Configuration
+      ##
+      # @return [String] Base path for dependents URLs.
+      #
       def base_path
         "#{Settings.evss.url}/wss-education-services-web/rest/education/chapter33/v1"
       end
 
+      ##
+      # @return [String] Service name to use in breakers and metrics.
+      #
       def service_name
         'EVSS/GiBillStatus'
       end
 
+      ##
+      # @return [Boolean] Should the service use mock data in lower environments.
```

Comment:
Will duplicate language in other documented service/**/configuration.rb files

Reply:
How would you suggest avoiding this duplication?

Reply:
Sorry I should have been more specific - I like this language and I think we should use it consistently for all of the service `configuration.rb` files. Probably didn't need to add the comment here :) :shipit: :fist_oncoming: